packages/@expo/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -129,8 +129,13 @@ export function getDefaultConfig(
   const metroConfig: Partial<MetroConfig> = mergeConfig(metroDefaultValues, {
     watchFolders,
     resolver: {
-      // unstable_conditionsByPlatform: { web: ['browser'] },
-      unstable_conditionNames: ['require', 'import', 'react-native'],
+      unstable_conditionsByPlatform: {
```

Comment:
Shouldn't this be "techincally" `unstable_conditionNamesByPlatform`, not sure if we want to keep the naming in sync with Metro's one. I'm fine either way 😄